setup.py

```diff
@@ -6,7 +6,7 @@
 
 INSTALL_REQUIRES = [
     'django-cms>=3.2.0',
-    'django>=1.8,<1.10',
+    'django>=1.8,<1.10.999',
```

Comment:
I was wondering about that, too. `django-filer` also uses `<1.10.999`:

https://github.com/divio/django-filer/blob/develop/setup.py#L29

Because *"Django is known to use rc versions"*. I guess it's to ensure a RC isn't installed ...